http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -336,20 +339,20 @@ object ClusterInitValues {
       dataprocConfig.rstudioServerName,
       dataprocConfig.welderServerName,
       proxyConfig.proxyServerName,
-      clusterRequest.jupyterUserScriptUri.map(_.toUri).getOrElse(""),
-      GcsPath(stagingBucket, GcsObjectName("userscript_output.txt")).toUri,
+      cluster.jupyterUserScriptUri.map(_.toUri).getOrElse(""),
+      cluster.dataprocInfo.stagingBucket.map(x => GcsPath(x, GcsObjectName("userscript_output.txt")).toUri).getOrElse(""),
       serviceAccountKey.map(_ => GcsPath(initBucketName, GcsObjectName(serviceAccountCredentialsFilename)).toUri).getOrElse(""),
-      userEmailLoginHint.value,
+      cluster.auditInfo.creator.value,
       contentSecurityPolicy,
-      clusterRequest.userJupyterExtensionConfig.map(x => x.serverExtensions.values.mkString(" ")).getOrElse(""),
-      clusterRequest.userJupyterExtensionConfig.map(x => x.nbExtensions.values.mkString(" ")).getOrElse(""),
-      clusterRequest.userJupyterExtensionConfig.map(x => x.combinedExtensions.values.mkString(" ")).getOrElse(""),
-      clusterRequest.userJupyterExtensionConfig.map(x => x.labExtensions.values.mkString(" ")).getOrElse(""),
+      cluster.userJupyterExtensionConfig.map(x => x.serverExtensions.values.mkString(" ")).getOrElse(""),
+      cluster.userJupyterExtensionConfig.map(x => x.nbExtensions.values.mkString(" ")).getOrElse(""),
+      cluster.userJupyterExtensionConfig.map(x => x.combinedExtensions.values.mkString(" ")).getOrElse(""),
+      cluster.userJupyterExtensionConfig.map(x => x.labExtensions.values.mkString(" ")).getOrElse(""),
       GcsPath(initBucketName, GcsObjectName(clusterResourcesConfig.jupyterNotebookConfigUri.value)).toUri,
       GcsPath(initBucketName, GcsObjectName(clusterResourcesConfig.jupyterNotebookFrontendConfigUri.value)).toUri,
-      clusterRequest.defaultClientId.getOrElse(""),
-      welderEnabled.toString,  // TODO: remove this and conditional below when welder is rolled out to all clusters
-      if (welderEnabled) dataprocConfig.welderEnabledNotebooksDir else dataprocConfig.welderDisabledNotebooksDir
+      cluster.defaultClientId.getOrElse(""),
+      cluster.welderEnabled.toString,  // TODO: remove this and conditional below when welder is rolled out to all clusters
```

Comment:
shall we change the comment to when we sunset firecloud...just so we don't forget about firecloud :)